packages/core-js-builder/README.md

```diff
@@ -32,3 +32,5 @@ const bundle = await builder({
   filename: PATH_TO_MY_COREJS_BUNDLE,
 });
 ```
+
+ℹ️ When using TypeScript, make sure to set `esModuleInterop` to `true`.
```

Comment:
I don't often use TypeScript, but IIRC with `export = ...` `esModuleInterop: true` is not required, or I missed something?

Reply:
It seems so:
![image](https://user-images.githubusercontent.com/2213682/210649632-a14f4fe3-addb-4d2d-9667-d9a87dd81cc4.png)

Reply:
If you omit that setting you get following error:
```
9 import builder from "core-js-builder"
         ~~~~~~~

  ../core-js/packages/core-js-builder/index.d.ts:60:1
    60 export = builder;
       ~~~~~~~~~~~~~~~~~
    This module is declared with 'export =', and can only be used with a default import when using the 'allowSyntheticDefaultImports' flag.
```

If you add `allowSyntheticDefaultImports` it compiles, but the compiled output is invalid as it looks like this (note the `.default`):
```js
const core_js_builder_1 = require("core-js-builder");
// ...
const bundle = await (0, core_js_builder_1.default)(this.options);
```

If you change the import to `import * as builder from 'core-js-builder'` the output is correct, so no `.default` is added, but you get following error:
```
src/plugin.ts:9:26 - error TS2497: This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export.

9 import * as builder from "core-js-builder"
                           ~~~~~~~~~~~~~~~~~
```
---
So the only way to import the module without errors is setting `esModuleInterop` to `true` and referencing the default export.
(I use TypeScript 4.9.4)